pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["flit_core ~=3.2"]
 build-backend = "flit_core.buildapi"
 
 [project]
-name = "universal-transfer-operator"
+name = "apache-airflow-provider-transfers"
```

Comment:
Do we want to change this?

Reply:
yes, otherwise it will upload as `universal-transfer-operator` in pypi